### PR TITLE
seastar-json2code: cleanups

### PR DESCRIPTION
**Pull request by tchaikov**
- seastar-json2code: remove unnecessary ';'
- seastar-json2code: remove redundant spaces
- seastar-json2code: s/x == True/x/
- seastar-json2code: do not look up in a dict twice

### CODE REVIEW

File: scripts/seastar-json2code.py [hunk 1]
```diff
@@ -52,10 +52,10 @@
 
 spacing = "    "
 def getitem(d, key, name):
-    if key in d:
+    try:
```
Comment (xemul) [inline scripts/seastar-json2code.py:55]: Maybe
```python
ret = d.get(key)
if ret:
    return ret
else:
    raise Exception(...)
```
?

**Comment by xemul** [comment]
All but the last patch look like fixing errors from some standard codechecker. Maybe it's then worth to equip the whole thing with e.g. `flake8` and put a config nearby so that the code is always compatible with the known set of rules? (_disclaimer_: not a python guy myself, don't know what the best practice here is)

File: scripts/seastar-json2code.py (hunk 1 above)
Comment (tchaikov) [reply scripts/seastar-json2code.py]: okay. changed as suggested. but i think more idiomatic way is to use exception for error checking.

**Comment by tchaikov** [comment]
> All but the last patch look like fixing errors from some standard codechecker.

yes, most of them were reported by flycheck integrated into my editor. and it uses flake8.

> Maybe it's then worth to equip the whole thing with e.g. `flake8` and put a config nearby so that the code is always compatible with the known set of rules? (_disclaimer_: not a python guy myself, don't know what the best practice here is)

i think the best way forward is to have a `tox.ini` which runs flake8 against this file. before that, we need to make sure flake8 does not complain. but i don't have enough bandwidth to do the cleanup in a single run. so probably we can do this piecemeal? and add the test when the whole program is PEP8 compliant?

FWIW, the reason why i started looking at this was because the uuid-based generation work. we expose it using the swagger API. and the json2code generator does not support `std::variant<int64_t, UUID>` yet.

**Pull request closed without merging**